lib/netconf/network.go

```diff
@@ -12,6 +12,18 @@ func (i ID) IsEphemeral() bool {
 	return i == Simnet || i == Devnet || i == Staging
 }
 
+// NodeSnapshotGethStateScheme returns the supported Geth node snapshot state scheme for the given network.
+func (i ID) NodeSnapshotGethStateScheme() string {
```

Comment:
Remember, AHA, if only used in one place, just put the func there.

But more important, we take archive node snapshots, they are always hash and never path in all networks.

Reply:
on staging that is not the case